merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/annotations/AutoValueMapper.java

```diff
@@ -0,0 +1,10 @@
+package gov.nasa.jpl.aerie.merlin.framework.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.CLASS)
+@Target(ElementType.TYPE)
+public @interface AutoValueMapper {}
```

Comment:
I agree with @pcrosemurgy comment on the name. `AutoValueMapper` sounds like it works for every kind of object. Maybe `RecordAutoValueMapper` ? Or we plan to generalize this auto value mapper ?